tests/test_fips.py

```diff
@@ -54,8 +49,29 @@
 RUN /bin/fips-test sha256
 """
 
+CONTAINER_IMAGES = []
+for param in ALL_CONTAINERS:
+    ctr, marks = container_and_marks_from_pytest_param(param)
+    print(marks)
+    if param in LTSS_BASE_FIPS_CONTAINERS:
+        CONTAINER_IMAGES.append(param)
+    else:
+        CONTAINER_IMAGES.append(
+            pytest.param(
+                ctr,
+                marks=marks
+                + [
+                    pytest.mark.skipif(
+                        not host_fips_enabled(),
+                        reason="The target must run in FIPS mode for the FIPS test suite",
+                    )
+                ],
```

Comment:
This + `tox -e format` please. Also, you can save the skip mark outside of the loop and reuse it (would improve readability a bit imho)
```suggestion
                marks=(marks
                + [
                    pytest.mark.skipif(
                        not host_fips_enabled(),
                        reason="The target must run in FIPS mode for the FIPS test suite",
                    )
                ]),
```

Reply:
Reformated :heavy_check_mark: